### DISCUSSION TITLE
Output error

### DISCUSSION THREAD

**Discussion by JianhuaQin** [exadg/exadg#60] [General] [answered]
Dear developers,

I tired to use exadg. It runs fine when I set "WriteOutput" to "false".
However, when this tag is changed to "true". Error occured.
Note that I have set "OutputDirectory" to "output", and I created an "output" directory in the same folder when the "solver" file exists.

Any clue to solve this problem is appreciated.

--Jianhua

**Comment by nfehn** [comment, marked as answer]
Thanks for your feedback!

Did you also create a `output/vtu` directory?

**Reply by nfehn** [reply]
Note that we also have an issue for this problem (issue #55), but I am pretty sure your problem can be solved in the meantime by creating all the directories manually.

**Reply by JianhuaQin** [reply]
> Thanks for your feedback!
> 
> Did you also create a `output/vtu` directory?

Many thanks for the immediate reply! Problem is solved now.
